### DISCUSSION TITLE
Documents type supported by saleor dasbboard while creating product?

### DISCUSSION THREAD

**Discussion by g-londhe** [saleor/saleor#10036] [Q&A] [answered]
What are the document types(jpg, png,...) supported for media while creating products, and is there any file size limit on the same?

**Comment by karolkielecki** [comment, marked as answer]
Hi @g-londhe, while creating a product you can upload both jpg and png images. You can also upload videos. The Saleor validation library supports the following services: Dailymotion, Flickr, Imgur, Instagram, MobyPicture, Photobucket, SmugMug, Scribd, TikTok, Twitter, Vimeo, WordPress.tv, YouTube. https://docs.saleor.io/docs/3.x/dashboard/catalog/products#step-7-add-media

**Reply by g-londhe** [reply]
Hi @karolkielecki,

We have a use case where we need to upload a pdf document for each product. We plan to use attributes for the same, however, the product attribute of file type does not support non-image type fields.

How do we add support for the same?

**Reply by karolkielecki** [reply]
File attribute allows you to upload pdf files as well. See my screenshot below 
![image](https://user-images.githubusercontent.com/57907951/175294217-60d3df9c-5343-4ae8-9657-348e1db9a7a0.png)
